index.html

```diff
@@ -14,55 +14,129 @@
             <div id="button-holder">
                 <img src="images/button.png" alt="Hamburger button" id="button">
             </div>
+            <div id="menubar">
+                <ul id="menu-list">
+                    <li><a class="anchor" href="#">Portfolio</a></li>
+                    <li><a class="anchor" href="#">About</a></li>
+                    <li><a class="anchor" href="#">Contact</a></li>
+                    <li><a class="anchor" href="#"><img src="images/mail.png" alt="Mail image"></a></li>
+                </ul>
+            </div>
         </nav>
     </header>
     <main id="content">
```

Comment:
It looks nice, but kindly make sure to decrease the height of this section, it'll make it much more beautiful.